repositories/noteRepository.js

```diff
@@ -0,0 +1,39 @@
+const Note = require('../models/Note');
```

Comment:
Same comment as above, let's rename this to note as the folder name suggests its content.

Reply:
Done